mdl/Dockerfile

```diff
@@ -1,4 +1,15 @@
 FROM iusdev.azurecr.io/web-build:latest AS pythonbuild
+
+# Nodejs
+ENV NODE_VERSION 10.18.0
```

Comment:
This should probably just be done in the base image: https://github.com/ius-gov/docker-images/blob/master/ius-build-base/Dockerfile

Reply:
thats the issue. i dont want to downgrade everyone, just the mdl image

Reply:
Oh, gotcha!